### ISSUE TITLE
Question about the scale of style loss and content loss

### ISSUE THREAD

**Issue by CodeYaCode** [gordicaleksa/pytorch-nst-feedforward#2]
Hi, gordicaleksa! This is a really impressive work. I have a question about the scale of style loss and content loss. I have implemented my own feedforward network, but the value of style loss and content loss are beyond 1e8 or 1e9, why the losses in your network are only 1~15?

**Comment by gordicaleksa** [comment]
Hi! Thanks! Sorry for the slow reply I didn't get the notification. 

Basically I experimented with it. I see over and over again people just copying other people's coefficients but I didn't like it as it seemed arbitrary. I think it has to do also with the fact that I avoided obscure divisions and normalization that some other implementations use, trying to make it as simple as possible. Only apply MSE with mean reduction and that's it.

This is especially true for my implementation of the original NST code (by Gatys et al.). Hope that answers your question!
